examples/net-fuzzing/README.md

```diff
@@ -0,0 +1,37 @@
+# getenv retrace fuzzing
+Retrace can be used to fuzz system calls related with networking.
+
+## getenv fuzz configuration syntax
+```
+fuzzing-net,[function name],[fuzzing type],[fuzzing rate]
+```
+## Function names and Fuzzing types:
+
+1. socket, accept
+NO_MEMORY, LIMIT_SOCKET
+
+2. connect
+ADDR_INUSE, NET_UNREACHABLE, CONN_TIMEOUT
+
+3. bind, listen
+ADDR_INUSE
+
+4. send, sendto, sendmsg
+CONN_RESET, NO_MEMORY
```

Comment:
@jjr840430 ok, but I think that we can understand that this fd is network or IO related and use `incompleteio` or `net-funzzing`.

But we can keep it as is.